sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-sources/files/5.15/z0008-net-mlx5-Free-IRQ-rmap-and-notifier-on-kernel-shutdo.patch

```diff
@@ -0,0 +1,113 @@
+From 0a5b94bd2b4775092acdbb077dcb2ea45dbf777e Mon Sep 17 00:00:00 2001
+From: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
```

Comment:
typically you want to preserve ownership so this line would be
```
From: Saeed Mahameed [<saeedm@nvidia.com>](mailto:saeedm@nvidia.com)
```
when you send it to the mailing list, the email will be from you but this will show up as the first line.

Like here: https://lore.kernel.org/stable/20230914015445.51657-1-sashal@kernel.org/T/#u (first example i found).